### ISSUE TITLE
'NoneType' Error in invoke_wmi

### ISSUE THREAD

**Issue by utkusen** [byt3bl33d3r/DeathStar#42]
I'm trying to use latest deathstar and empire but it seems the old bug is still there. It's hard to define the exact problem since it sometimes work, sometimes doesn't. But let me write the issue here, maybe someone can explain it better.

```
[-] Agent: K4AX2UV9 => Error executing module 'powershell/lateral_movement/invoke_wmi': {'error': 'module produced an empty script'}
Exception in thread Thread-2:
Traceback (most recent call last):
  File "/usr/local/Cellar/python3/3.6.0/Frameworks/Python.framework/Versions/3.6/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "deathstar.py", line 58, in __run
    self.__run_backup()
  File "/usr/local/Cellar/python3/3.6.0/Frameworks/Python.framework/Versions/3.6/lib/python3.6/threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "deathstar.py", line 529, in spread
    invoke_wmi(agent_name, box)
  File "deathstar.py", line 454, in invoke_wmi
    results = execute_module_with_results('powershell/lateral_movement/invoke_wmi', agent_name, module_options)
  File "deathstar.py", line 179, in execute_module_with_results
    if entry['taskID'] == r['taskID']:
TypeError: 'NoneType' object is not subscriptable
```

Please note that old version of deathstar with old version of empire (taskid workaround applied) works without any problem for me

**Comment by DanMcInerney** [comment]
This is fixed in my PR

**Comment by byt3bl33d3r** [comment]
Resolved in #50.

Closing
